app/helpers/events_helper.rb

```diff
@@ -16,4 +16,19 @@ def users_for_event(state)
     requests = requests.where(state: state) if state != 'all'
     requests.distinct.pluck('users.email')
   end
+
+  def state_label(state)
```

Comment:
Just return the css class and do the rest in the view because it is always the same